app/components/list-item/index.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from '@testing-library/react'
+import ListItem from '.'
+import { ListItemType } from '@/app/app-types'
+import { faker } from '@faker-js/faker'
+
+describe('List-item component: test cases', () => {
+  const items: ListItemType[] = [
+    {
+      id: faker.string.uuid(),
+      name: faker.word.words(),
+    },
+  ]
+  test('render list-item component', () => {
+    render(<ListItem items={items} />)
+    const listItemEl = screen.getByTestId('list-item-wrapper')
+    expect(listItemEl).toBeInTheDocument()
+  })
+
+  test('load the list elements', () => {
+    const listItems: ListItemType[] = [
+      {
+        id: faker.string.uuid(),
+        name: faker.word.words(),
+      },
+      {
+        id: faker.string.uuid(),
+        name: faker.word.words(),
+      },
+    ]
+    render(<ListItem items={listItems} />)
+    const listItemEl = screen.getAllByRole('listitem')
+    expect(listItemEl.length).toBe(2)
+  })
+
+  test('label is rendered with the right value', () => {
```

Comment:
You can cover the negative test cases also when there are no records in the list

Reply:
Covered 'no-records' test case.